user/mixins.py

```diff
@@ -0,0 +1,22 @@
+from django.core.exceptions import PermissionDenied
+
+
+class GroupRequiredMixin(object):
+    """
+    group_required - list of strings, required param
+    """
+
+    group_required = None
```

Comment:
It's best practice to make it not List, but Tuple to prevent this attribute to be changed during the runtime